src/com/goide/dlv/protocol/DlvApi.java

```diff
@@ -94,6 +98,10 @@
     public Function function;
   }
 
+  public static class StacktraceOut {
```

Comment:
For reference this is how the API looks like right now: https://github.com/dlsniper/delve/blob/alphabetically-sort-server-methods/service/rpc2/server.go (it's not the complete version as more changes where merged after I've made this commit)